clang/tools/driver/cc1_main.cpp

```diff
@@ -198,13 +199,21 @@ static int PrintSupportedExtensions(std::string TargetStr) {
   std::unique_ptr<llvm::TargetMachine> TheTargetMachine(
       TheTarget->createTargetMachine(TargetStr, "", "", Options, std::nullopt));
   const llvm::Triple &MachineTriple = TheTargetMachine->getTargetTriple();
+  const llvm::MCSubtargetInfo *MCInfo = TheTargetMachine->getMCSubtargetInfo();
+  const llvm::ArrayRef<llvm::SubtargetFeatureKV> Features =
+    MCInfo->getAllProcessorFeatures();
+
+  std::map<StringRef, StringRef> llvmDescMap;
```

Comment:
Potentially this could be https://llvm.org/doxygen/classllvm_1_1StringMap.html instead.